sky/packages/sky/lib/src/widgets/drag_target.dart

```diff
@@ -78,39 +79,37 @@ class _DraggableState extends State<Draggable> {
         break;
     }
     assert(dragStartPoint != null);
-    _route = new DragRoute(
+    _avatar = new _DragAvatar(
       data: config.data,
       dragStartPoint: dragStartPoint,
       feedback: config.feedback,
       feedbackOffset: config.feedbackOffset,
       onDragFinished: () {
-        _route = null;
+        _avatar = null;
       }
     );
-    _route.update(point);
-    Navigator.of(context).push(_route);
```

Comment:
Yeah, the old code is busted too, but for different reasons.  We should re-think what we're trying to do with ephemeral routes.